pkg/client/client_test.go

```diff
@@ -234,7 +226,10 @@ func testObjects(t *testing.T) {
 
 	assert.Equal(t, nil, err)
 	assert.Equal(t, []string{"schema", "name", "type", "owner", "comment"}, res.Columns)
-	assert.Equal(t, []string{"public"}, mapKeys(objects))
+	_, ok := objects["public"]
+	if !ok {
+		t.Fail()
+	}
```

Comment:
@matoubidou  Thank for raising this PR. I have the same usage that needs to show empty schemes.

I think it's better to maintain the pattern using assert by doing

```go
assert.Contains(t, objects, "public")
```

Reply:
Thx.... changed it.